basicHTML - vi.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" name="viewport" content="width=device-width, initial-scale=1.0">
+		<title> Sing Up </title>
+	</head>
+	<body>
+		<center>
+		<div align="center" style="background-color:#F2F5A9; padding:20px; width:400px">
+		<h2> Sign Up </h2>
+		
+		<div align="left">
+		<form method="post" action="signUp">
+			<fieldset>
+			<legend>Enter your personal information</legend>
+			<p> 
+				<label> First Name </label> </br> <input type="text" name="username" required placeholder="First name" />	
```

Comment:
Yo should use the 'for' attribute in the labels, it's a formal way to link a label to a form input.
See the example below:
```
<div>
    <label for="name">Name:</label>
    <input type="text" id="name" name="user_name">
  </div>
```